xcube/cli/prune.py

```diff
@@ -33,22 +33,22 @@
 # noinspection PyShadowingBuiltins
 @click.command(name='prune')
 @click.argument('dataset_path', metavar='DATASET')
-@click.option('--verbose', '-v', is_flag=True, multiple=True,
+@click.option('--verbose', '-v', count=True,
               help='Verbose mode. Multiple may be given, '
                    'for example "-vvv".')
 @click.option('--dry-run', is_flag=True,
               help='Just read and process input, '
                    'but don\'t produce any output.')
 def prune(dataset_path: str,
-          verbose: Tuple[bool, ...],
+          verbose: int,
           dry_run: bool):
     """
     Delete empty chunks.
     Deletes all data files associated with empty
     (NaN-only) chunks in given DATASET,
     which must have Zarr format.
     """
-    max_verbosity = sum(verbose)
+    max_verbosity = verbose
```

Comment:
Rename into `verbosity` and delete line.

Reply:
Not sure about that, because the whole code is: 

```
def prune(dataset_path: str,
          verbosity: int,
          dry_run: bool):
    """
    Delete empty chunks.
    Deletes all data files associated with empty
    (NaN-only) chunks in given DATASET,
    which must have Zarr format.
    """
    max_verbosity = verbosity

    def monitor(msg: str, verbosity: int = 1):
        if verbosity <= max_verbosity:
            print(msg)

    _prune(input_path=dataset_path, dry_run=dry_run, monitor=monitor)
    return 0
```

Maybe it would be good to change the monitor funktion into: 

```
def prune(dataset_path: str,
          verbosity: int,
          dry_run: bool):
    """
    Delete empty chunks.
    Deletes all data files associated with empty
    (NaN-only) chunks in given DATASET,
    which must have Zarr format.
    """

    def monitor(msg: str, verbosity: int):
        if verbosity:
            print(msg)

    _prune(input_path=dataset_path, dry_run=dry_run, monitor=monitor)
    return 0
```

Reply:
Does this work? `verbosity` in the `monitor` callback is a keyword-argument whose name should not be changed. Renaming it to `monitor_verbosity` may break the callback contract. Please verify.